### PR TITLE
Move image focal point app to marketplace

### PR DESCRIPTION
**Pull request by giotiskl**

### CODE REVIEW

File: marketplace/image-focal-point/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+4. Click on `Publish` in the sidebar to publish the new image with focal point entry.
+
+### Step 3: Configure your content type that needs a reference to the image with focal point
+
+To make use of the newly created image with focal point entry:
+
+1. Go to the `Content model` page
+2. Edit the content type that needs to reference the image with focal point
+3. Create a new field of type `Reference` and open its configuration.
+4. Set a validation option to accept only entries of type `Image with Focal Point`
```
Comment (davidfateh) [inline marketplace/image-focal-point/README.md]: I don't think we should use the world 'only' as it isn't necessary for it to be exclusive.
```suggestion
4. Set a validation option to accept entries of type `Image with Focal Point`
```

File: marketplace/image-focal-point/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+<img src="./assets/apps-image-focal-point-validations.png" width="500" alt="Set validation rule"/>
+
+5. Optionally set the widget of the new reference field to `Entry Card` to have a preview of the image with focal point it links to
```
Comment (davidfateh) [inline marketplace/image-focal-point/README.md]:
```suggestion
5. Optionally, set the widget of the new reference field to `Entry Card` to have a preview of the image with focal point it links to
```

File: marketplace/image-focal-point/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+<img src="./assets/apps-image-focal-point-entry-card-widget.png" width="500" alt="Set Entry Card widget"/>
+
+6. Go back to the `Content` page
+7. Create/edit an entry of the content type referencing the image with focal point.
+8. Set the reference field you added to point to the image with focal point entry.
+
```
Comment (davidfateh) [inline marketplace/image-focal-point/README.md]: In all the numbered lists in this file, you sometimes exclude an ending period. I would add them all in for consistency.

**Comment by jelz** [comment]
@giotiskl I'd implement David's suggestions and also apply them in the docs.

**Pull request merged**